docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Create a Tile
```

Comment:
Why does this page have the Datadog Sites dropdown? It doesn't affect anything does it?

Reply:
@alai97

Reply:
They all have the Datadog Sites dropdown, it doesn't affect anything assuming that the content is the same throughout all data centers.